### DISCUSSION TITLE
aerospace + stage manager?

### DISCUSSION THREAD

**Discussion by onexbash** [nikitabobko/AeroSpace#793] [questions-and-answers]
Hi together, 
The docs describe that the intention of aerospace is to be used with 1 space per monitor and the built-in spaces feature to be ignored. 
What about the stage manager? Is it possible / does it make sense to use aerospace together with the macos stage manager, or does this contradict the intended function of aerospace?

**Comment by refleraga** [comment]
i would think they clash with each other, yes. imo aerospace and stage manager manage windows very similarly in that they store groups of windows (aerospace: workspaces, stage manager: grouped apps) that we can switch to and fro within a single space, but:

- aerospace is keyboard-centric, whereas stage manager at the moment is fully keyboard-noninteractive.
- stage manager supports multiple spaces, whereas aerospace's workspaces are meant to be a replacement for spaces and thus doesn't work properly with more than one space (per monitor).
